Sources/OpenTelemetryApi/Common/AttributeValue.swift

```diff
@@ -11,10 +11,7 @@ public enum AttributeValue: Equatable, CustomStringConvertible, Hashable {
   case bool(Bool)
   case int(Int)
   case double(Double)
-  case stringArray([String])
```

Comment:
My concern is this breaking change. Should I try to retain these attributes and have them translate to AttributeArray under the hood, or do we rev the main version indicating a breaking change? @nachoBonafonte